airflow/config_templates/config.yml

```diff
@@ -454,23 +454,23 @@
       description: |
         The folder where airflow should store its log files
         This path must be absolute
-      version_added: ~
+      version_added: 2.0.0
```

Comment:
All logging configuration options have been moved to a new section in Airflow 2.0, so they do not work on older versions.